src/snowflake/snowpark/session.py

```diff
@@ -760,16 +817,9 @@ def close(self) -> None:
                 _remove_session(self)
 
     @property
-    def conf(self) -> RuntimeConfig:
+    def conf(self) -> SettingStore:
```

Comment:
we should probably keep the name same for migration sake: https://spark.apache.org/docs/latest/api/python/reference/pyspark.sql/api/pyspark.sql.conf.RuntimeConfig.html